### DISCUSSION TITLE
Insert 3D model

### DISCUSSION THREAD

**Discussion by manuelkoch98** [StrandedKitty/streets-gl#187] [Q&A]
Is it possible to replace a 3D model? In other words, to replace my own model of my house with the default 3D model?

**Comment by pktiuk** [comment]
Hi @manuelkoch98    
You don't directly import 3D models into maps.   
You can just map buildings using OpenStreetMap editor and in this editor select height, type of roof, colour of building etc.
https://wiki.openstreetmap.org/wiki/Simple_3D_Buildings
